CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#258](https://github.com/realm/SwiftLint/issues/258)
 
+* `ForceUnwrappingRule` was added as an `OptInRule` to issue warnings for all forced unwrappings
```

Comment:
Please adjust the formatting of your changelog entry to match https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes.

* Period at the end
* Two trailing spaces
* Include the issue number, not the PR number